hca/upload/client_side_checksum_handler.py

```diff
@@ -0,0 +1,53 @@
+import os
+import time
+
+from dcplib.checksumming_io import ChecksummingSink
+from dcplib.s3_multipart import get_s3_multipart_chunk_size
+
+# Checksum(s) to compute for file; current options: crc32c, sha1, sha256, s3_etag
+CHECKSUM_NAMES = ['crc32c']
+
+
+class ClientSideChecksumHandler:
+    """ The ClientSideChecksumHandler takes in a file as a parameter and handles any behavior related to
+    check-summing the file
+    on the client-side, returning a tag that can be used as metadata when the file is uploaded to S3."""
+
+    def __init__(self, filename):
+        self._checksums = {}
+        self._filename = filename
+
+    def getChecksumMetadataTag(self):
```

Comment:
Any reason for camel case vs standard python underscore naming?

Reply:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Nope... just habit.